src/main/java/core/basesyntax/Machines/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Machines;
+
+public class Excavator extends Machine{
+
+  @Override
+  public void doWork() {
+    System.out.println("Excavator is excavating something.");
```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working. While you have specified 'Excavator', it would be more informative to include a specific action or task it's performing, if that's applicable to the context of the application.